src/main/java/spoon/support/visitor/java/JavaReflectionVisitorImpl.java

```diff
@@ -43,9 +43,9 @@ public void visitPackage(Package aPackage) {
 			clazz.getPackage();
 		}
 		if (clazz.getSuperclass() != null) {
```

Comment:
is it ok that the check is not on `getGenericSuperclass`?

Reply:
Yeah I certainly should change that one for safety.